src/scss/constants.scss

```diff
@@ -90,6 +90,8 @@ $spacing--lg: 16px;
 $spacing--xl: 24px;
 $spacing--xxl: 30px;
 
+$iframe-box-shadow: 2px 2px $spacing--lg $white;
+
```

Comment:
Hrmm.. still not sure this is right? The only file I expect to see any changes from on this PR should be in `src/scss/customOverrides.scss`?